gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PubsubSubscriptionController.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.controllers;
+
+import com.netflix.spinnaker.gate.services.PubsubSubscriptionService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class PubsubSubscriptionController {
+
+  @Autowired
+  private PubsubSubscriptionService pubsubSubscriptionService;
+
+  @ApiOperation(value = "Retrieve the list of pub/sub subscriptions configured in Echo.")
+  @RequestMapping(value="/pubsub/subscriptions", method = RequestMethod.GET)
```

Comment:
Spacing is inconsistent with the other annotation parameters here - `value = "..."`